legal-api/src/legal_api/models/filing.py

```diff
@@ -682,6 +712,16 @@ def get_previous_completed_filing(filing):
             return filings[0]
         return None
 
+    @staticmethod
+    def get_filings_sub_type(filing_type: str, filing_json: dict):
+        """Return sub-type from filing json if sub-type exists for filing type."""
+        filing_sub_type_key = Filing.FILING_SUB_TYPE_KEYS.get(filing_type)
+        if filing_sub_type_key:
+            filing_sub_type = filing_json['filing'][filing_type][filing_sub_type_key]
+            return filing_sub_type
```

Comment:
So what does the restoration schema look like now? Do you have a sample filing?

Reply:
Well, not much BE work has been done besides opening it up to save at this point I think.  A minimal restoration schema for full and limited previously existed.  Probably for ops work.  

I only see full restorations in prod.  The filing json looks like this 

``` json

{
  "filing": {
    "header": {
      "date": "2016-09-09",
      "name": "restoration",
      "effectiveDate": "2021-09-14T08:00:00-00:00",
      ...
    },
    "business": {
      ...
    },
    "restoration": {
      "date": "2021-09-14",
      "type": "fullRestoration"
    }
  }
}

```

I'll try creating one in Dev for you.

Reply:
OK, thanks for the clarity.

I've already created some in Dev for me :)